tests/datasource/fluent/test_databricks_sql_datasource.py

```diff
@@ -0,0 +1,71 @@
+from __future__ import annotations
+
+import pydantic
+import pytest
+
+from great_expectations.datasource.fluent.databricks_sql_datasource import (
+    DatabricksSQLDatasource,
+)
+
+
+@pytest.mark.unit
+@pytest.mark.parametrize(
+    "connection_string, expected_errors",
+    [
+        pytest.param(
+            "databricks+connector://token:my_token>@my_host:1234/my_db",
+            [
+                {
+                    "loc": ("connection_string",),
+                    "msg": "ConfigStr - contains no config template strings in the format '${MY_CONFIG_VAR}' or '$MY_CONFIG_VAR'",
+                    "type": "value_error",
+                },
+                {
+                    "loc": ("connection_string",),
+                    "msg": "URL query invalid",
+                    "type": "value_error.url.query",
+                },
+            ],
+            id="missing query",
+        ),
+        pytest.param(
+            "databricks+connector://token:my_token>@my_host:1234/my_db?my_query=data",
+            [
+                {
+                    "loc": ("connection_string",),
+                    "msg": "ConfigStr - contains no config template strings in the format '${MY_CONFIG_VAR}' or '$MY_CONFIG_VAR'",
+                    "type": "value_error",
+                },
+                {
+                    "loc": ("connection_string",),
+                    "msg": "HTTP path invalid",
```

Comment:
```suggestion
                    "msg": "'http_path' query param is invalid or missing",
```
Would this be a better error message?